frontend/ui/src/components/Templates/RenderTemplate.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {TextTemplate} from './TextTemplate';
+//import {Template} from 'httpclient';
+
+type Props = {
+  styleVariant?: 'small' | 'normal';
+  template: any;
+};
+
+const RenderTemplate = ({styleVariant, template}: Props) => {
+  const templateContent = JSON.parse(template.content) as any;
+  switch (templateContent.blueprint) {
+    case 'text':
+      return <TextTemplate styleVariant={styleVariant} template={templateContent} isMember={true} />;
+
+    default:
+      return null;
+  }
+};
+
+export default RenderTemplate;
```

Comment:
yeah so I know it makes sense the render library to render the templates, but I guess we would need to tweak it to accept something else than messages